doc/reference/tooling/tt_cli/export.rst

```diff
@@ -89,15 +104,62 @@ In this case, field values start from the second row, for example:
 Exporting compound data
 -----------------------
 
-By default, ``tt`` exports empty values for fields containing compound data such as arrays or maps.
+In the CSV format, ``tt`` exports empty values by default for fields containing compound data such as arrays or maps.
 To export compound values in a specific format, use the ``--compound-value-format`` option.
-For example, the command below exports compound values serialized in JSON:
+For example, the command below exports compound values to CSV serialized in JSON:
 
 .. code-block:: console
 
     $ tt crud export localhost:3301 customers:customers.csv  \
                      --compound-value-format json
 
+.. _tt-export-tdg2:
+
+Exporting from Tarantool Data Grid 2
+------------------------------------
+
+The command below exports data of the ``customers`` space from a TDG2 cluster to
+the ``customers.jsonl`` file:
+
+.. code-block:: console
+
+    $ tt tdg2 export localhost:3301 customers:customers.jsonl
+
+If the ``customers`` space has four fields (``id``, ``firstname``, ``lastname``, and ``age``), the file with exported data might look like this:
+
+.. code-block:: json
+
+    {"age":30,"first_name":"Samantha","id":1,"second_name":"Carter"}
+    {"age":41,"first_name":"Fay","id":2,"second_name":"Rivers"}
+    {"age":74,"first_name":"Milo","id":4,"second_name":"Walters"}
+
+If a tuple contains a ``null`` value in a field, this field is not exported:
+
+.. code-block:: json
+
+    {"age":13,"first_name":"Zachariah","id":3}
+
+Tuple fields that contain maps with non-string keys are converted maps with string keys.
+
+
+TDG2 sets a limit on the number of tuples returned in result of a query execution
```

Comment:
> in result of a query execution

It is the limit for per-storage response for an internal request between router and storage. Theoretically, a user can receive up to `shards_count` * `hard-limit.returned` records, but, since sharding distribution is hard to predict beforewise, it is also theoretically possible to fail on a hard limit for any request that returns more than `hard-limit.returned`.